src/wasm.cc

```diff
@@ -36,13 +37,60 @@ namespace proxy_wasm {
 
 namespace {
 
-// Map from Wasm Key to the local Wasm instance.
+// Map from Wasm key to the thread-local Wasm instance.
 thread_local std::unordered_map<std::string, std::weak_ptr<WasmHandleBase>> local_wasms;
+// Wasm key queue to track stale entries in `local_wasms`.
+thread_local std::queue<std::string> local_wasms_keys;
+
+// Map from plugin key to the thread-local plugin instance.
 thread_local std::unordered_map<std::string, std::weak_ptr<PluginHandleBase>> local_plugins;
+// Plugin key queue to track stale entries in `local_plugins`.
+thread_local std::queue<std::string> local_plugins_keys;
+
+// Check no more than `max_local_cache_gc_chunk_size` cache entries at a time during stale entries
+// cleanup.
+const size_t max_local_cache_gc_chunk_size = 64;
```

Comment:
Since proxy-wasm-cpp-host uses [Envoy coding style](https://github.com/envoyproxy/envoy/blob/main/STYLE.md),  use PascalCase or ALL_CAPS for constants.

Reply:
Sure, done. Thanks!